0001-basic-expiry-model.md

```diff
@@ -0,0 +1,94 @@
+
+## Research
+
+### Findings
+
+@LLFourn and I had a couple of meetings and we figured out some things:
+
+- The difference between `alpha_expiry` and `beta_expiry` has an effect on the security of the model for *Bob*. This does not affect Alice since she will have already redeemed `beta_asset` by the time this is relevant.
+- `beta_expiry` has an effect on the time that *Alice* will have to redeem `beta_asset` and therefore on the likelihood that the protocol will be completed successfully. This does _not_ have any direct effect on the security of the protocol, but rather on its *execution guarantees*.
+
+Given this, the topic of this spike changed from creating a security model to, more generally, creating a basic model to calculate expiry times.
+
+### Alpha expiry
+
+There are two ways in which `alpha_expiry`, <img src="https://latex.codecogs.com/gif.latex?E_{\alpha}"/>, has an effect on the protocol:
+
+1. How long Bob will (at least) have to redeem `alpha_asset` if Alice redeems `beta_asset`.
+2. How long Alice will have to wait to be able to refund `alpha_asset` if the protocol isn't carried out to completion.
+
+It would therefore be ideal to find a value of `alpha_expiry` that lets Bob redeem `alpha_asset` with high probability while also ensuring Alice doesn't have to wait too long to be able to be refunded if it comes to that.
+
+#### Assumptions
+
+- Bob provides <img src="https://latex.codecogs.com/gif.latex?T_{\alpha}"/>, the time in which he is overwhelmingly confident he will be able to confirm a transaction on `alpha_ledger`. He has direct influence over this since he can choose the fee.
+- Bob provides <img src="https://latex.codecogs.com/gif.latex?k"/>, the number of additional confirmations he would need to have on his redeem transaction before he considers `alpha_asset` to be his own.
+- Block time on `alpha_ledger` can be modelled as a random variable that follows an exponential distribution with parameter <img src="https://latex.codecogs.com/gif.latex?\lambda"/>.
+- A confirmation is given by the arrival of a new block.
+- The value of `beta_expiry`, <img src="https://latex.codecogs.com/gif.latex?E_{\beta}"/>, is constant.
+
+#### Model
+
+To determine `alpha_expiry`, the time <img src="https://latex.codecogs.com/gif.latex?\Delta_{R}"/> Bob will need to get his desired number of confirmations on his redeem transaction of `alpha_asset` will need to be estimated.
+
+The time it would take for Bob's redeem transaction to obtain <img src="https://latex.codecogs.com/gif.latex?k"/> confirmations on `alpha_ledger` can be modelled as a sum of <img src="https://latex.codecogs.com/gif.latex?k"/> mutually independent exponential random variables with the same rate parameter <img src="https://latex.codecogs.com/gif.latex?\lamda"/>, which is equal to an <img src="https://latex.codecogs.com/gif.latex?\textrm{Erlang}(k,\lambda)"/> distribution.
```

Comment:
```suggestion
The time it would take for Bob's redeem transaction to obtain <img src="https://latex.codecogs.com/gif.latex?k"/> confirmations on `alpha_ledger` can be modelled as a sum of <img src="https://latex.codecogs.com/gif.latex?k"/> mutually independent exponential random variables with the same rate parameter <img src="https://latex.codecogs.com/gif.latex?\lambda"/>, which is equal to an <img src="https://latex.codecogs.com/gif.latex?\textrm{Erlang}(k,\lambda)"/> distribution.
```